qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -651,6 +652,11 @@ def is_recently_audible(self):
         page = self._widget.page()
         return page.recentlyAudible()
 
+    def _on_url_changed(self):
+        urlstr = self._tab.url().toString(QUrl.RemoveUserInfo)
+        muted_url = config.instance.get('content.mute', url=urlstr)
```

Comment:
This is a bit weird. I think you might be confusing the 'per domain settings' feature with 'reading a list of URLs' (as mentioned in the docstring). 

Instead of checking if muted_url is not none here, I think what would make more sense, is a `content.audible_default` (or similarly named) setting, which defaults to true. Then, you could just read that value and set it here. 

If people want to turn it off per domain (via `--pattern`), they can do so, since you're passing `url` to the `get` function. If people want to selectively enable it, they can do the opposite (set it false globally, and true per-url).